taxcalc/calculate.py

```diff
@@ -322,6 +326,8 @@ def mtr(self, variable_str='e00200p',
             self.records.e00900 = seincome_var + finite_diff
         elif variable_str == 'e00650':
             self.records.e00600 = divincome_var + finite_diff
+        elif variable_str == 'e26270':
+            self.records.e02000 = schEincome_var + finite_diff
```

Comment:
@codykallen Why add finite_diff to e02000 instead of e26270 here?  Seems like adding to e26270 is what would help this MTR enter `functions.SchXYZ()` in a way that accounts for potential differences between rates on ordinary income and pass-through income.